app/src/main/res/drawable/ic_sqrl_icon_into_safe_vector_outline.xml

```diff
@@ -0,0 +1,64 @@
+<vector android:height="24dp" android:viewportHeight="2400"
```

Comment:
Where did we find these safe icons? Did you create them? I'm just asking because of copyrights.

Reply:
I drew them myself using the Inkscape tool.

Reply:
Great work, you are definitely better at incscape than me. Tried to add a shadow to the button on the Simplified view and failed horribly so I reverted that thought and the new look is better anyway :)